setup/setup_all.sh

```diff
@@ -113,7 +114,7 @@ init_foundation() {
 # Build all microservices and deploy to the cluster:
 deploy_microservices_to_gke() {
   cd "$BASE_DIR"/terraform/stages/gke
-  terraform init -backend-config=bucket="${TF_BUCKET_NAME}"
+  terraform init -reconfigure -backend-config=bucket="${TF_BUCKET_NAME}"
```

Comment:
We should consider using `terraform init -chdir` and execute it from the base directory

Reply:
Good idea, let's tackle this in later PR because it will require multiple changes across many files. Can you raise a Github issue as a FR for this?